Chapter7/shapes.erl

```diff
@@ -0,0 +1,19 @@
+-module(shapes).
+-export([perimeter/1, area/1]).
+
+-record(circle, {radius}).
+-record(rectangle, {length, width}).
+-record(triangle, {side1, side2, side3}).
+
+% Exercise 7-4: Records and Shapes
+
+perimeter( #circle{radius=Radius} ) 							-> 2 * math:pi() * Radius;
+perimeter( #rectangle{length=Length, width=Width} ) 			-> (2 * Length) + (2 * Width);
+perimeter( #triangle{side1=Side1, side2=Side2, side3=Side3} ) 	-> Side1 + Side2 + Side3.
+
+
+area( #circle{radius=Radius} ) 								-> math:pow(math:pi() * Radius, 2);
```

Comment:
Github ate all formatting, please use [preformatted](https://guides.github.com/features/mastering-markdown/) markdown when you want to paste code :)
I assume in your case, since it's the same line, there should not be any alignments.